android/src/main/java/com/rt2zz/reactnativecontacts/ContactsManager.java

```diff
@@ -37,9 +43,15 @@ public void getAll(Callback callback) {
     ContentResolver cr = getReactApplicationContext().getContentResolver();
     Uri uri = ContactsContract.Contacts.CONTENT_URI;
     String selection = ContactsContract.Contacts.IN_VISIBLE_GROUP + " = " + 1;
+    Resources resources = this.reactContext.getResources();
     String sortBy = CommonDataKinds.Contactables.LOOKUP_KEY;
-
-    Cursor cursor = cr.query(uri, null, selection, null, sortBy);
+    String [] projection = new String[]{
+            CommonDataKinds.Contactables._ID,
+            CommonDataKinds.Contactables.DISPLAY_NAME,
```

Comment:
I think I've got a fix - https://github.com/rt2zz/react-native-contacts/pull/48

Have tested but only on older versions of android (and am going on holiday very soon so might not have time to test properly for a week or so).

Anyway - looks like it gets the data back fine.  I'll at least try and do some performance testing tomorrow.